pre_commit/languages/docker.py

```diff
@@ -78,6 +79,17 @@ def install_environment(
 
 def get_docker_user() -> Tuple[str, ...]:  # pragma: win32 no cover
     try:
+        output = subprocess.check_output(
```

Comment:
This new block shouldn't be inside try/except

Reply:
Where should I put it to avoid the `AttributeError` error like: https://asottile.visualstudio.com/asottile/_build/results?buildId=3829&view=logs&j=ae542f9d-96d9-59d2-9a06-a2a85a17a811&t=6deecb4d-4f67-5e8d-1829-1f5cd5833176

Reply:
That is a part of the test. The code expects `os.getuid()` or `os.getgid()` raise that exception and you make it expect that your new code may also raise that exception which is not true. But even if it was, you shouldn't put more than one line inside of a try/except block because the expectations would be too broad.

Reply:
OMG! I did that?

Reply:
Most likely. Just move it outside, before `try`.

Reply:
Done this. If I manage to get something useful out of this PR, I will owe some beers to half of the python planet ...